app/controllers/public/index.js

```diff
@@ -113,13 +113,6 @@ export default Controller.extend({
         return;
       }
       let { order, event } = this.model;
-      let tax =  await event.get('tax');
-      if (tax) {
-        if (!tax.get('isTaxIncludedInPrice')) {
```

Comment:
@uds5501 Why did you remove the following ?

Reply:
@kushthedude because it was adding tax on tax. I will take care of this

Reply:
@kushthedude accidentally resolved the other one. The ticket fees is something which organisers have to handle, not the ticket buyers. It's  more like a cut for the platform (@CosmicCoder96 please correct me if I am wrong)

Reply:
> @kushthedude because it was adding tax on tax. I will take care of this

Removing this will break the order summary page . As it was responsible for tax modal display in order wizard

Reply:
@kushthedude it won't. I double checked that. you can go ahead and try it in this PRs deployment itself.

Reply:
@uds5501 Is it sending the amount back to server? Because last time it was added here was because it was not reflecting the taxed amount on server and everything was correct on frontend Please check it again that after order too correct amount is getting saved both on server and FE.

Reply:
see this https://github.com/fossasia/open-event-frontend/pull/2916#issuecomment-492743708

Reply:
This looks good since now `total` on the ticker list itself contains the taxed amount.